.sphinx/lxd-doc/README.md

```diff
@@ -0,0 +1,172 @@
+# lxd-doc
+
+A small CLI to parse comments in a Golang codebase meant to be used for a documentation tool (like Sphinx for example).
+
+It parses the comments from the AST and extracts their documentation.
+
+# Usage
+
+    $ lxd-doc -h
+    Usage of lxd-doc:
+    -e value
+        Path that will be excluded from the process
+
+# Formatting
+
+A comment is formatted this way:
+
+```go
+	// lxddoc:generate(group=cluster, key=scheduler.instance)
+	//
+	//  <Possibly a very long documentation on multiple lines with Markdown tables, etc.>
+	// ---
+	//  shortdesc: Possible values are all, manual and group. See Automatic placement of instances for more information.
+    //  condition: container
+	//  default: `all`
+	//  type: integer
+	//  liveupdate: `yes`
```

Comment:
Now the <ANY_KEY>: <ANY_VALUE> line is missing?
(And the indentation is messed up again :wink: )

Reply:
@ru-fu somehow, on my side, the `<ANY_KEY>: <ANY_VALUE>` is pproperly indented but on github it is different...

Reply:
Ok I managed :) (vscode tab config was weird)

Reply:
Thanks, all good now! :)